doc/development_process.md

```diff
@@ -0,0 +1,185 @@
+- If the behaviour is intended, we might consider why people are frustrated by it.  Is player-end feedback a problem?  Is the UI creating tedium?  This is an area where things can languish for a while, but it doesn't mean we don't care, it's just that these problems are hard to fix.  This is also where the experimental/stable thing becomes relevant.  For example, we will often leave a feature in experimental that makes the game unpleasant, because we do want that feature, and we want it working as intended.  On a few occasions, we've patched the behaviour out and then put it back in after stable.  This is to preserve gameplay for the people playing the curated, stable version of the game, while also continuing to work on the features we want to include.
+
+#### Why this format?  Why not finish things before putting them in a public experimental release, or revert them if they're broken?
+
+Various forms of this sentiment are really common, and there are dozens of different good reasons.  Let's look at just one, though, because it's possibly the most important and also the most difficult to see unless you really study how development works in the long term.
+
+Allowing things to get broken in experimental allows us to have big, complex projects that move the game forward in leaps and bounds.  If we forced ourselves to keep experimental "fun" all the time, contributors wouldn't be free to try new things and risk causing big play balance changes.  We'd never have monster weak points.  We'd never have customizable sidebars.  We probably wouldn't NPCs at all, possibly not even Z-levels.  We wouldn't even be able to do code infrastructure changes that cause impacts on players, because things like the transition to one-second turns or the removal of charges have been woefully unpleasant to play through as we sort them out.  We'd also be less welcoming to new, less experienced contributors, even if we didn't want to be, and in the long term we wouldn't have a better player experience to show for it, because many of our most interesting changes come about as the product of long - often years long - projects that remain in a frustrating, partially complete state as we sort them out.  If we were a professional full-time team, these things would probably progress a bit faster, but we work on volunteer time, and this is the system we've found that allows volunteers to develop systems that would be at home in a professional game, while having only a few hours a week of their time to contribute.
```

Comment:
```suggestion
Allowing things to get broken in experimental allows us to have big, complex projects that move the game forward in leaps and bounds.  If we forced ourselves to keep experimental "fun" all the time, contributors wouldn't be free to try new things and risk causing big play balance changes.  We'd never have monster weak points.  We'd never have customizable sidebars.  We probably wouldn't have NPCs at all, possibly not even Z-levels.  We wouldn't even be able to do code infrastructure changes that cause impacts on players, because things like the transition to one-second turns or the removal of charges have been woefully unpleasant to play through as we sort them out.  We'd also be less welcoming to new, less experienced contributors, even if we didn't want to be, and in the long term we wouldn't have a better player experience to show for it, because many of our most interesting changes come about as the product of long - often years long - projects that remain in a frustrating, partially complete state as we sort them out.  If we were a professional full-time team, these things would probably progress a bit faster, but we work on volunteer time, and this is the system we've found that allows volunteers to develop systems that would be at home in a professional game, while having only a few hours a week of their time to contribute.
```

Reply:
The change is: "wouldn't NPCs" -> "wouldn't have NPCs"

Reply:
Ah thanks. I have already shortened this chunk a lot so that sentence got cut